gokart/parameter.py

```diff
@@ -5,11 +5,21 @@
 import luigi
 from luigi import task_register
 
+import gokart
+
 logger = getLogger(__name__)
 
 
 class TaskInstanceParameter(luigi.Parameter):
 
+    def __init__(self, *args, **kwargs):
```

Comment:
Does following works as the same manner?

```
def __init__(self, bound=gokart.TaskOnKart, *args, **kwargs):
```

Reply:
Thank you for the suggestion. 

It doesn't work due to the following error. Maybe it's because  the default value is evaluated in `__init__.py` before initializing `TaskOnKart` (TaskInstanceParameter is imported in `task.py`).

```AttributeError: partially initialized module 'gokart' has no attribute 'TaskOnKart' (most likely due to a circular import)```